java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java

```diff
@@ -43,7 +43,7 @@
     Addresses xpubAddr = deriveAddresses(address, publishConnection);
     Addresses xsubAddr = deriveAddresses(address, subscribeConnection);
 
-    LOG.info(String.format("XPUB binding to %s, XSUB binding to %s", xpubAddr, xsubAddr));
+    LOG.finest(String.format("XPUB binding to %s, XSUB binding to %s", xpubAddr, xsubAddr));
```

Comment:
This change is unhelpful: it precludes users from knowing which ports are being used for what purpose within the system.